internal/e2e/node/module_resolution.spec.js

```diff
@@ -7,6 +7,8 @@ const node_resolve_main_2 = require('node_resolve_main_2');
 const node_resolve_nested_main = require('node_resolve_nested_main');
 const lib1 = require('lib1');
 const lib1some = require('lib1/src/some');
+const data_resolve_relative = require.resolve('./data/data.json')
```

Comment:
hmm we don't have a linter here for semicolons I guess

Reply:
Fixed